packages/suite/src/views/wallet/nfts/NftsTable/NftsTable.tsx

```diff
@@ -0,0 +1,93 @@
+import { SelectedAccountLoaded } from '@suite-common/wallet-types';
+import { Card, Column, Table, H3 } from '@trezor/components';
+import { selectNftDefinitions } from '@suite-common/token-definitions';
+
+import { Translation } from 'src/components/suite/Translation';
+import { useSelector } from 'src/hooks/suite';
+import { getNfts } from 'src/utils/wallet/nftUtils';
+
+import NftsRow from './NftsRow';
+import { filterNftTokens } from '@suite-common/wallet-utils';
+import { Token } from '@trezor/blockchain-link-types/src/blockbook-api';
+import { getNetwork } from '@suite-common/wallet-config';
+
+type NftsTableProps = {
+    selectedAccount: SelectedAccountLoaded;
+    searchQuery: string;
+    type: 'ERC721' | 'ERC1155';
+    shown?: boolean;
+    verified?: boolean;
+};
+
+const NftsTable = ({ selectedAccount, searchQuery, type, shown, verified }: NftsTableProps) => {
+    const { account } = selectedAccount;
+    const nftDefinitions = useSelector(state => selectNftDefinitions(state, account.symbol));
+    const filteredTokens = filterNftTokens(account?.tokens || []);
+    const nfts = getNfts({ tokens: filteredTokens, symbol: account.symbol, nftDefinitions });
+    const network = getNetwork(account.symbol);
+
+    const filterNftsByType = (nfts: Token[]) => {
+        return nfts.filter(nft => nft.type === type);
+    };
+
+    const getNftsToShow = () => {
+        if (shown) {
+            return [...nfts.shownVerified, ...nfts.shownUnverified];
+        }
+
+        return verified ? nfts.hiddenVerified : nfts.hiddenUnverified;
+    };
+
+    const nftsToShow = getNftsToShow();
+    const filteredNfts = filterNftsByType(nftsToShow);
+
+    if (filteredNfts.length === 0) {
+        return null;
+    }
+
+    return (
+        <Column width="100%" alignItems="start" gap={12}>
+            <H3>{type}</H3>
+            <Card paddingType="none" overflow="hidden">
+                <Table>
+                    <Table.Header>
+                        <Table.Row>
+                            <Table.Cell colSpan={1}>
+                                <Translation id="TR_COLLECTION" />
```

Comment:
Collection name

I would not shorten name of Collection + please use BlurUrls
![Screenshot 2024-11-23 at 18 09 38](https://github.com/user-attachments/assets/d780da69-2f01-49b1-9a1e-7820d283fb39)